test/schema/match/artist.js

```diff
@@ -0,0 +1,71 @@
+import schema from "../../../schema"
+import { runQuery } from "../../utils"
+
+describe("Me", () => {
+  describe("MatchArtist", () => {
+    let gravity
+
+    const MatchArtist = schema.__get__("MatchArtist")
+
+    beforeEach(() => {
+      gravity = sinon.stub()
+      MatchArtist.__Rewire__("gravity", gravity)
+    })
+
+    afterEach(() => {
+      MatchArtist.__ResetDependency__("gravity")
+    })
+
+    it("returns bidder ids that the user is registered in sales for", () => {
+      const query = `
+        {
+          match_artist(term: "ok") {
+            id
+            name
+            birthday
+          }
+        }
+      `
+      const response = [
+        {
+          id: "han-myung-ok",
+          name: "Han Myung-Ok",
+          birthday: "1958",
+          artworks_count: 12,
+        },
+      ]
+      gravity.withArgs("match/artists", { term: "ok" }).returns(Promise.resolve(response))
+
+      return runQuery(query).then(data => {
+        expect(data).toEqual({ match_artist: [{ birthday: "1958", id: "han-myung-ok", name: "Han Myung-Ok" }] })
+      })
+    })
+  })
+})
+
+// console.log("OK?")
```

Comment:
yeah, I forgot that in order to use jest mocks I needed to remove all instances of babel-rewire - see https://github.com/artsy/metaphysics/pull/635 for that stone to roll